src/constants/index.tsx

```diff
@@ -650,6 +652,9 @@ export const RoutablePlatformKeysByNetwork = {
   [ChainId.BSC_TESTNET]: [],
 }
 
+/**
+ * @TODO check why we have 2 different UNISWAP routable platforms
+ */
```

Comment:
We have 2 Uniswap routable platforms, one for V2 and one for V3 ,although I don't know if they V2 one is still being used. Good "question" Rorry.